### DISCUSSION TITLE
How to replicate custom configuration across multiple machines?

### DISCUSSION THREAD

**Discussion by alwindoss** [NvChad/NvChad#2323] [Q&A]
Nvchad expects all custom configuration to exist within the custom directory. However, if I want to replicate my configuration across multiple devices how do I go about storing configuration in Git?
One that I can think of if I am not using Git is copy the custom directory to the machine where I want to replicate. But, if I am using Git is there a cleaner way to do this?

**Comment by benzwt** [comment]
if your machines support NFS, it is the best solution.
or sshfs.

**Comment by siduck** [comment]
@alwindoss  i'd just put my dotfiles repo ( with custom dir ) in my home folder and then symlink it to ~/.config/nvim

**Comment by alwindoss** [comment]
Is there anyway git submodules can be used to achieve this?

**Reply by siduck** [reply]
@alwindoss maybe, give it a try
